src/olympia/conf/prod/settings.py

```diff
@@ -161,15 +124,11 @@
 UGLIFY_BIN = 'uglifyjs'
 ADDONS_LINTER_BIN = 'addons-linter'
 
-LESS_PREPROCESS = True
-
 XSENDFILE_HEADER = 'X-Accel-Redirect'
 
-GEOIP_URL = 'https://geo.services.mozilla.com'
+ALLOW_SELF_REVIEWS = False
 
-AES_KEYS = env.dict('AES_KEYS')
-
-SENTRY_DSN = env('SENTRY_DSN')
+GEOIP_URL = 'https://geo.services.mozilla.com'
```

Comment:
I can't find anything where this is being used, neither in our installed dependencies nor in olympia itself. Might be a good candidate to get rid of

Reply:
There's a quite a few random settings I couldn't find referenced in our code, nor in `settings_base.py`.  A future task might be to reorder them in an import-style - i.e. django settings at the top, then 3rd party, then ours (and sorted alphabetically) to make it slightly easier to work out what's used/needed where.